src/Application/Application.php

```diff
@@ -80,10 +82,14 @@ public function run(array $inputArgs): int
                 return $executionExitCode;
             }
         } catch (ScriptNotFoundException $e) {
-            $this->cliMate->red()->bold("Script with name {$inputArgs[1]} not found\n");
+            $this->notifyError("Script with name {$inputArgs[1]} not found\n");
         }
 
-        $this->showListing($scriptFinder->getAllScripts());
+        try {
+            $this->showListing($scriptFinder->getAllScripts());
+        } catch (ScriptPathNotValidException $e) {
+            $this->notifyError($e->getMessage() . "\n");
```

Comment:
That doesn't seem right. You are `catching` an exception here but the result code is `SUCCESS`.
For every catched exception an `ERROR` result should be returned.

```
$this->notifyError();
return self::RESULT_ERROR;
```